### PR TITLE
CI: use Composer stable release for AppVeyor

### PR DESCRIPTION
**Pull request by kubawerlos**

### CODE REVIEW

**Comment by kubawerlos** [comment]
@keradus @SpacePossum @julienfalque should we use Composer stable version and go with this PR or wait for Composer snapshot to be fixed?

**Comment by julienfalque** [comment]
I see no reason to prefer snapshot release over stable, let's keep stable.

File: .appveyor.yml [hunk 1]
```diff
@@ -29,13 +29,14 @@ install:
...
-    - IF NOT EXIST C:\tools\composer.phar (cd C:\tools && appveyor DownloadFile https://getcomposer.org/composer.phar)
+    - IF NOT EXIST C:\tools\composer.phar (cd C:\tools && appveyor DownloadFile https://getcomposer.org/composer-stable.phar)
+    - copy composer-stable.phar composer.phar
     - cd C:\projects\php-cs-fixer
     - php C:\tools\composer.phar global show hirak/prestissimo -q || php C:\tools\composer.phar global require hirak/prestissimo
 
 before_test:
     - cd C:\projects\php-cs-fixer
-    - php C:\tools\composer.phar update --optimize-autoloader --no-interaction --no-progress --prefer-stable --no-ansi
+    - php C:\tools\composer.phar update --optimize-autoloader --no-interaction --no-progress --prefer-stable --no-ansi -vvv --profile
```
Comment (kubawerlos) [inline .appveyor.yml:39]:
```suggestion
    - php C:\tools\composer.phar update --optimize-autoloader --no-interaction --no-progress --prefer-stable --no-ansi
```

Comment (kubawerlos) [inline .appveyor.yml:39]:
```suggestion
    - php C:\tools\composer.phar update --optimize-autoloader --no-interaction --no-progress --prefer-stable --no-ansi
```

**Comment by keradus** [comment]
Thank you @kubawerlos.

**Pull request merged**